Wancer/Wancer/Views/MessagePage.swift

```diff
@@ -0,0 +1,144 @@
+import SwiftUI
```

Comment:
To reiterate what we discussed last meeting, we do not need preview to work. If you are having trouble with connecting to database in the ViewModel, just pr the View.